addons/binding/org.openhab.binding.enocean/src/main/java/org/openhab/binding/enocean/internal/discovery/EnOceanUsbSerialDiscoveryParticipant.java

```diff
@@ -76,7 +77,9 @@ private ThingUID createBridgeThingType(UsbSerialDeviceInformation deviceInformat
 
     private boolean isEnoceanUSB300Dongle(UsbSerialDeviceInformation deviceInformation) {
         return deviceInformation.getVendorId() == ENOCEAN_USB300_DONGLE_VENDOR_ID
-                && deviceInformation.getProductId() == ENOCEAN_USB300_DONGLE_PRODUCT_ID;
+                && deviceInformation.getProductId() == ENOCEAN_USB300_DONGLE_PRODUCT_ID
+                && deviceInformation.getManufacturer() != null
+                && deviceInformation.getManufacturer().equalsIgnoreCase(ENOCEAN_USB300_DONGLE_MANUFACTURER);
```

Comment:
Maybe also add a check for `getProduct()` ? 

I see it returns something useful [here](https://knx-user-forum.de/forum/supportforen/smarthome-py/868516-enocean-auf-raspberry-2):

```

[    2.538720] usb 1-1.4: new full-speed USB device number 4 using dwc_otg
[    2.641254] udevd[175]: starting version 175
[    2.667448] usb 1-1.4: New USB device found, idVendor=0403, idProduct=6001
[    2.670040] usb 1-1.4: New USB device strings: Mfr=1, Product=2, SerialNumber=3
[    2.672227] usb 1-1.4: Product: EnOcean USB 300 DA
[    2.674455] usb 1-1.4: Manufacturer: EnOcean GmbH
[    2.676741] usb 1-1.4: SerialNumber: FTWMW7SQ
```

Reply:
Hi @wborn,

I did not add a check for ```getProduct()``` as you can find at least two different product types (DA and DB) in the net. Mine is a ```EnOcean USB 300 DB```. However I could add a contains ```USB 300``` check. Will update the PR tomorrow.

Reply:
Hi @wborn,

I updated the check and included the product now too.

Best regards
Daniel